Monika After Story/game/script-topics.rpy

```diff
@@ -15269,3 +15269,66 @@ label monika_being_herself:
     m 5eka "With you, I can allow myself to be...{w=0.5}{nw}"
     extend 5eua "Well, {i}Just Me.{/i}"
     return
+    
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_remembrance",
+            category=['monika'],
+            prompt="How much of your past do you remember?",
+            conditional="seen_event('monika_japan')",
+            pool=True
+        )
+    )
+
+label monika_remembrance:
+    m 1esc "I'm not really sure myself, [player]..."
+    m 2rsc "It's weird because I do have memories of my past, but as soon as I start to gather my thoughts they become kind of...{w=0.5}{nw}"
+    extend 2esd "fuzzy."
+    m 7eksdlc "Like, I won't be able to recall where something took place, or the names of the people who were there..."
+    m 7dksdld "...I have memories of growing up in a normal neighborhood, but beyond that, I can't even tell you what my parent's faces look like..."
+    m 1rkc "It's all like...{w=0.5}feelings and impressions."
+    m 3wud "What's even weirder is not everything's like this!"
+
+    if seen_event('monika_wine') and not persistent._mas_pm_cares_about_dokis:
+        m 1rusdla "There are things I remember perfectly, like that one time Yuri brought wine to the clubroom...{w=0.3}or Sayori wolfing down the whole cookie jar."
```

Comment:
Is it the Sayori bit itself that bothers you, or do you want to cut it for pacing? I think it sounds better with 2 examples, could replace it with `...{w=0.3}or how Nasuki sometimes brought cupcakes.` or something.

Reply:
Sayori bit itself isn't ever mentioned in DDLC or by Monika, so its accuracy is questionable. Pacing as well I felt it could be shorter, especially since the condition of it itself is about the Yuri's Wine topic, so keeping the line purely related to that makes the most sense